plugins/godot-gradle-plugin/src/main/kotlin/godot/gradle/subplugin/GodotSubPlugin.kt

```diff
@@ -52,7 +53,7 @@ class GodotSubPlugin : KotlinGradleSubplugin<AbstractCompile> {
         //&& task is KotlinNativeCompile -> we cannot do this check here as the compiler plugin get's called in the
         // linking stage again, which then fails as the subPlugin is disabled for it and thus does not provide the
         // needed command line arguments
-        project.plugins.hasPlugin(GodotPlugin::class.java)
+        project.extensions.findByType<GodotExtension>() != null
```

Comment:
I totally forgot about this, always check for the extension not the plugin class. This can cause issue when you are checking for a presence of your plugin in the configuration phase.